docs/coins.md

```diff
@@ -35,6 +35,7 @@ This list is generated from [./coins.json](../coins.json)
 | 313     | Zilliqa          | ZIL    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/zilliqa/info/logo.png" width="32" />      | <https://zilliqa.com>         |
 | 330     | Terra            | LUNA   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/terra/info/logo.png" width="32" />        | <https://terra.money>         |
 | 354     | Polkadot         | DOT    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/polkadot/info/logo.png" width="32" />     | <https://polkadot.network/>   |
+| 394     | Crypto_org       | CRO    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/cryptoorg/info/logo.png" width="32" />    | <https://crypto.org>          |
```

Comment:
Logo PNG is 404, please create PR to add blockchains/cryptoorg/info files to assets repository.

Reply:
ok, The PR was #8805 but I was not sure of the order. don't want to pay for "nothing" :/

Reply:
I have adjusted and merged the assets PR.
https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/cryptoorg/info/logo.png